clumper/clump.py

```diff
@@ -147,6 +150,14 @@ def read_csv(cls, path, delimiter=",", fieldnames=None, n=None):
                         the correct number of fieldnames is supplied, as an incorrect number can lead
                         to truncation. If you have seven columns and your fieldnames length is 3,
                         then every row will have only 3 values, the remaining four will be cut off.
+            na_values:  Null values. If `ignore`, null values are passed as is. If `None`, then for each row,
```

Comment:
> null values are passed as is 

That means we'll get empty strings or `None` values? Might be good to emphasize that.